dotcom-rendering/package.json

```diff
@@ -38,12 +38,12 @@
 		"@guardian/identity-auth": "6.0.1",
 		"@guardian/identity-auth-frontend": "8.1.0",
 		"@guardian/libs": "23.0.0",
-		"@guardian/ophan-tracker-js": "2.2.10",
+		"@guardian/ophan-tracker-js": "2.3.0",
 		"@guardian/react-crossword": "6.3.0",
 		"@guardian/shimport": "1.0.2",
 		"@guardian/source": "9.0.0",
 		"@guardian/source-development-kitchen": "18.1.1",
-		"@guardian/support-dotcom-components": "7.4.0",
+		"@guardian/support-dotcom-components": "7.5.0",
```

Comment:
support-dotcom-components has a dependency on @guardian/ophan-tracker-js so this bump is to keep both packages in line.